nio/block/base.py

```diff
@@ -85,7 +86,7 @@ def configure(self, context):
         # verify that block properties are valid
         self.validate()
 
-        self.logger = get_nio_logger(self.name())
+        self.logger = get_nio_logger(self.label(True))
```

Comment:
Would prefer the logger have no ID if a name is specified. This will keep logs cleaner and if they don't specify a name it will still have the ID. The only time the ID would help would be if they somehow had two block configs with the same name but different ID. Unlikely scenario